content/2016-12-26_komunitni-python-stanek-na-linuxdays.md

```diff
@@ -0,0 +1,90 @@
+
+![Koncert pro Chris Ward a jeho podcastu o LinuxDays]({filename}/images/linuxdays_img_2389.jpg){:width="350px" :center}
+
+Pokud Tě zajímá nějaké pěkné shrnutí LinuxDays 2016, kdo tam byl a nebyl a co se tam dělo, rozhodně si pusť tenhle [super podcast od Chrise z Brna](https://soundcloud.com/chris-ward-908424795/rozhovory-na-linuxdayscz-2016-radio-beat-brno-petr-simacek).♪♫♪♪♫
+
+A k našemu stánku. Tak v první řadě jsme nikdy předtím nic takového nedělali. A tím to možná byl super uvolněný punk, který se nám náramně povedl a na který jsme i patřičně hrdí.
+Při plánování došlo nejprve na suché záležitosti typu letáčky, samolepky. Nutno míti na každém správném stánku. Sběratelé samolepek, zdá se, ještě nevyhynuli.
+Ovšem s velkým nápadem přišel Jakub Vysoký, jakožto hrdý syn svého otce Jardy Vysokého z Roudnice nad Labem, navrhl zapojit i kluky z tamějšího Gymnázia a jejich učitele Jardu.
+Což byl opravdu skvělý tah. Když už se nám ráno před samotnou akcí povedlo shromáždit vše a všechny, zjistili jsme, že jsme jeden z největších stánků co do rozlohy i do počtu prezentujících.
+Na stánku jsme totiž měli několik her na zelených destičkách typu NodeMCU, micro:bit a na nich hry, které poháněl MicroPython.
+O tyto destičky byl velký zájem a tak tímto velmi děkuji Petru Viktorinovi, Lumíru Balharovi a Janu Čermákovi, že mě v tom nenechali a všem nadšencům ochotně vysvětlovali a flashovali jejich destičky.
+
+Zmínění Roudničáci si přivezli her asi 10 a kdyby jim večer nevypli proud, sedí tam ještě teď. Na Adamovi, Jakubovi a Jardovi z Roudnice je super, že dokážou okolí nejen pobavit, ale i proškolit a aby někomu nepřišlo druhý den, že mají stejný program, tak 
+kluci přes noc ještě pár her doprogramují. Skvělým příkladem toho byla hra rádoby „GTAčko“, ke které přes noc Adam kromě dosavadního volantu s akcelerátorem ještě pedál fungující na světla z auta nalezená na půdě.
+Velké nadšení vzbudila jejich hra s vlaječkami na rychlost, digitální tabule na kterou se dá kreslit či hrát had dálkovým ovládáním. Rádoby „GTAčko“ s pedálem, volantem a světly. Jednoznačně parádní show a u našeho stánku byl téměř pořád plno.
+
+![Rádoby "GTAčko"]({filename}/images/linuxdays_img_2194.jpg){:width="350px" :center}
+
+![Hry na stánku měly velký úspěch]({filename}/images/linuxdays_img_2211.jpg){:width="350px" :center}
+
+![Jarda Vysoký s digitální tabulí, kterou naprogramoval s klukama z Gymnázia Roudnice nad Labem]({filename}/images/linuxdays_img_2216.jpg){:width="350px" :center}
+
+
+Při té příležitosti jsme lidem povídali o tom, co vše v české Python komunitě děláme. Překvapivé pro nás bylo, že se na konferenci objevilo spousty lidí, kteří nikdy ani o python.cz neslyšeli, na Pyvo srazech nebyli a celkově byli nadšení, že se o tom mohli dozvědět, protože v Pythonu dělají.
+Dále jsme samozřejmě prezentovali kromě python.cz i PyLadies, Advanced PyLadies, Blog Python komunity, PyCon v Brně.
```

Comment:
odkazy, všude odkazy! na článek dám odkaz z Twitteru nebo FB a čtenář si nebude dohledávat kde co je, když to nebude znát